workspaceitem-data-cclicense.md

```diff
@@ -0,0 +1,74 @@
+# WorkspaceItem data of CC License sectionType
+[Back to the definition of the workspaceitems endpoint](workspaceitems.md)
+
+The section data represents the data about the CC license
+
+```json
+{
+  	"granted": true,
```

Comment:
no need to have a granted attribute

Reply:
the presence of the uri does indeed display whether it's granted as well